### ISSUE TITLE
QGIS 3.10.11  crash after install in Debian Buster

### ISSUE THREAD

**Issue by giedrioks26** [qgis/QGIS#39967]
when i run qgis the terminal shows an error:
:~$ qgis
QGIS died on signal 11 (aborted)

**Comment by gioman** [comment]
@giedrioks26 installed how?

**Comment by giedrioks26** [comment]
Yes. Half an hour ago

**Comment by gioman** [comment]
> Yes. Half an hour ago

@giedrioks26 self compiled? using repositories? if using repositories, what repositories?

**Comment by giedrioks26** [comment]
I always use this https://qgis.org/debian-ltr

**Comment by giedrioks26** [comment]
same with 3.16.0 from https://qgis.org/debian and 3.17 from https://qgis.org/debian-nightly
After install 3.17 terminal shows:

`:~$ qgis
../../src/app/main.cpp:510 : (main) [0ms] Starting qgis main
../../src/core/qgsmessagelog.cpp:29 : (logMessage) [47ms] 2020-11-11T19:40:59 [1] Successfully loaded globalsettingsfile path: /usr/share/qgis/resources/qgis_global_settings.ini
../../src/core/qgsproviderregistry.cpp:258 : (init) [406ms] Loaded 25 providers (DB2;OAPIF;WFS;arcgisfeatureserver;arcgismapserver;delimitedtext;gdal;geonode;gpx;grass;grassraster;mdal;memory;mesh_memory;mssql;ogr;ows;pdal;postgres;postgresraster;spatialite;vectortile;virtual;wcs;wms) 
../../src/providers/grass/qgsgrass.cpp:518 : (lock) [2ms] lock
../../src/providers/grass/qgsgrass.cpp:317 : (init) [0ms] do init
../../src/providers/grass/qgsgrass.cpp:2660 : (defaultGisbase) [0ms] gisbase from envar = 
../../src/providers/grass/qgsgrass.cpp:2686 : (defaultGisbase) [0ms] gisbase = /usr/lib/grass76
../../src/providers/grass/qgsgrass.cpp:483 : (isValidGrassBaseDir) [0ms] isValidGrassBaseDir()
../../src/providers/grass/qgsgrass.cpp:2660 : (defaultGisbase) [0ms] gisbase from envar = 
../../src/providers/grass/qgsgrass.cpp:2686 : (defaultGisbase) [0ms] gisbase = /usr/lib/grass76
../../src/providers/grass/qgsgrass.cpp:383 : (init) [0ms] Valid GRASS gisbase is: /usr/lib/grass76
../../src/providers/grass/qgsgrass.cpp:2660 : (defaultGisbase) [0ms] gisbase from envar = 
../../src/providers/grass/qgsgrass.cpp:2686 : (defaultGisbase) [0ms] gisbase = /usr/lib/grass76
../../src/providers/grass/qgsgrass.cpp:2660 : (defaultGisbase) [0ms] gisbase from envar = /usr/lib/grass76
../../src/providers/grass/qgsgrass.cpp:2660 : (defaultGisbase) [0ms] gisbase from envar = /usr/lib/grass76
../../src/providers/grass/qgsgrass.cpp:419 : (init) [0ms] sGrassModulesPaths = /usr/lib/grass76/bin,/usr/lib/grass76/scripts,/usr/share/qgis/grass/scripts,/usr/lib/qgis/grass/modules
../../src/providers/grass/qgsgrass.cpp:524 : (unlock) [2ms] unlock
../../src/core/qgsellipsoidutils.cpp:271 : (ellipsoidParameters) [43ms] setEllipsoid: no row in tbl_ellipsoid for acronym 'NONE'
QGIS died on signal 11 (aborted)`

**Comment by gioman** [comment]
@giedrioks26 just tried on a clean Debian buster machine, all ok here.

**Comment by giedrioks26** [comment]
> just tried on a clean Debian buster machine, all ok here.

Got it. This will probably be my debian problem. Thanks for your time.

**Comment by giedrioks26** [comment]
I reinstalled debian on two difference pc, but the problem is still there. From debian back-ports also remains. Only works QGIS 2.18 version from the debian repository. This happened after yesterday's QGIS 3.10.11 updates.
Has anyone yet faced that?

**Comment by gioman** [comment]
> I reinstalled debian

@giedrioks26 from scratch? and still got the issue? this is weird, it is the same I did yesterday to check this ticket.

**Comment by giedrioks26** [comment]
> from scratch? and still got the issue? this is weird, it is the same I did yesterday to check this ticket.

@gioman after installation you upgraded with apt dist-upgrade?

**Comment by gioman** [comment]
> @gioman after installation you upgraded with apt dist-upgrade?

@giedrioks26 yes

**Comment by giedrioks26** [comment]
> yes

very strange ... I had no other choice, I had to upgrade debian from buster to unstable. Now it works.

**Comment by giedrioks26** [comment]
I suspect why this happened:
1. When I first upgraded qgis to 3.10.11, qgis was running at that time.
2. After the shutdown, the restart failed.
3. I moved the Home directory to another computer with debian sid. I started QGIS. He started. I turned it off. And I returned the Home directory to the debian buster again. Now everything works.

What could have happened when qgis was updated while it was running?
